src/test/java/nl/thehyve/transmartvariantstoreconnector/service/FilteringObservationClientServiceTests.java

```diff
@@ -131,12 +151,17 @@ public void givenSimpleConstraint_whenQueryObservations_thenStatus200() throws E
             .content("{\"type\":\"clinical\",\"constraint\":{\"type\":\"study_name\",\"studyId\":\"Study Z\"}}"))
             .andExpect(status().isOk())
             .andExpect(content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON))
-            .andExpect(jsonPath("$.dimensionDeclarations[0].name", is("study")))
-            .andExpect(jsonPath("$.cells", hasSize(4)))
-            .andExpect(jsonPath("$.cells[3].stringValue", is("male")));
+            .andDo(result -> {
+                Thread.sleep(100);
+                Hypercube hypercube = objectMapper.readValue(result.getResponse().getContentAsByteArray(), Hypercube.class);
+                Assert.assertEquals("study", hypercube.getDimensionDeclarations().get(0).getName());
+                Assert.assertThat(hypercube.getCells(), hasSize(4));
+                Assert.assertEquals("male", hypercube.getCells().get(3).getStringValue());
+            });
     }
 
     @WithMockUser(username="spring")
+    @Ignore
```

Comment:
What is wrong with this one?

Reply:
Thanks for noticing. I removed the `@Ignore` now.